nntrainer/models/neuralnet.cpp

```diff
@@ -101,7 +101,7 @@ NeuralNetwork::NeuralNetwork(AppContext app_context_) :
 
 int NeuralNetwork::loadFromConfig(const std::string &config) {
   if (loadedFromConfig == true) {
-    ml_loge("cannnot do loadFromConfig twice");
+    ml_loge("can not do loadFromConfig twice");
```

Comment:
Is this change made on purpose?

Reply:
Yes, I found a typo while modifying the code.

Reply:
@DonghakPark https://dictionary.cambridge.org/us/dictionary/english/cannot

[Cannot ](https://www.merriam-webster.com/grammar/cannot-vs-can-not-is-there-a-difference) is far more common and is therefore recommended.